tests/utils_tests/test_timesince.py

```diff
@@ -140,3 +140,42 @@ def test_thousand_years_ago(self):
         t = datetime.datetime(1007, 8, 14, 13, 46, 0)
         self.assertEqual(timesince(t, self.t), '1000\xa0years')
         self.assertEqual(timeuntil(self.t, t), '1000\xa0years')
+
+    def test_depth(self):
+        t = self.t + self.oneyear + self.onemonth + self.oneweek + self.oneday + self.onehour
+        self.assertEqual(timesince(self.t, t, depth=5), '1\xa0year, 1\xa0month, 1\xa0week, 1\xa0day, 1\xa0hour')
```

Comment:
You can merge `test_depth()`, `test_depth_out_of_range()`, and `test_depth_one()` into a single test `test_timesince_depth()` and use `self.subTest()`.